### PR TITLE
Adding Negation Evaluators

### PR DESCRIPTION
**Pull request by samriddhi99**
PR to add some negated evaluators.

- [x] Classes for negated evaluators
- [x] Tests for newly added evaluators
- [ ] Code is commented and well formatted

### CODE REVIEW

File: src/tirith/core/evaluators/not_contains.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+logger = logging.getLogger()
+
+# Checks if :attr: `evaluator_input` is contained in :attr:`evaluator_data`.
+
+# Args:
+#     value (mixed): Value to compare.
+#     other (mixed): Other value to compare.
+
+# Returns:
+#     bool: Whether :attr:`value` is equal to :attr:`other`.
```
Comment (refeed) [inline src/tirith/core/evaluators/not_contains.py:16]: Change the comment

File: src/tirith/core/evaluators/not_contains.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+#     >>> eq(None, None)
+#     False
+#     >>> eq(None, '')
+#     True
+#     >>> eq('a', 'a')
+#     False
+#     >>> eq(1, str(1))
+#     True
+
+# .. versionadded:: 1.0.0-alpha.1
```
Comment (refeed) [inline src/tirith/core/evaluators/not_contains.py:29]: Update to the current version

**Review by refeed — COMMENTED** [review]
Squash the commit into one

**Review by refeed — APPROVED** [review]
LGTM

**Comment by refeed** [comment]
(Related to https://github.com/StackGuardian/tirith/issues/101)

**Review by arunim2405 — APPROVED** [review]

**Pull request merged**